src/main/kotlin/com/cognifide/gradle/aem/common/instance/service/groovy/GroovyConsole.kt

```diff
@@ -15,7 +15,20 @@ import java.io.File
  */
 class GroovyConsole(sync: InstanceSync) : InstanceService(sync) {
```

Comment:
Consider using delegation instead of inheritance.

Reply:
This is to make code more simple in services impl